controllers/reviews.controller.js

```diff
@@ -27,3 +29,15 @@ exports.getCommentsById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postComment = (req, res, next) => {
+  const { review_id } = req.params;
+  const comment = req.body;
+  checkIdExists(review_id).catch((err) => next(err));
```

Comment:
review_id is also a foreign key so again, a separate check that the review exists should not be necessary as should also throw a PSQL error and also if it is not a number